build-data-pipeline-ffi.sh

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
+
+# Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+
+destdir="$1"
+
```

Comment:
do we want to fix the build logics before adding the tracing logics or after ?

Reply:
Could you please elaborate a bit on what this would mean.

Reply:
This is relative to the RFC + issue https://github.com/DataDog/libdatadog/issues/353
The idea would be to have a single build where we can select what is required to build.
A good first step could be to unify the build scripts.
Ideally all logics would be embedded in cargo in the long term.
cargo build --feature trace-pipeline,profiling,telemetry

Reply:
Fixed in the latest commit. As discussed internally profiling library will be the main artifact and data-pipeline symbols will be exported there.

Reply:
Why do we export the symbols here ?

Reply:
Forgot the last commit (pushed it but the prompt asked for the credentials while I was AFK) now it's complete.